packages/fluentui/docs/src/components/ComponentDoc/ComponentExample/ComponentExample.tsx

```diff
@@ -224,6 +224,10 @@ class ComponentExample extends React.Component<ComponentExampleProps, ComponentE
     return this.kebabExamplePath;
   };
 
+  getSourceFileName = () => {
+    return `Source code: ${this.props.examplePath.split('/').pop()}.tsx`;
```

Comment:
This does not handle shorthand examples and the problem is that some examples have only the shorthand variant.
So instead of blindly appending `.tsx` I would keep it without that (just `ExampleName`) and the user needs to know that it is the name without optional `shorthand` and without `tsx`.